torch/jit/__init__.py

```diff
@@ -1011,5 +1011,15 @@ def _find_builtin(fn):
     return _get_builtin_table().get(id(fn))
 
 
+class _DisableTracing(object):
```

Comment:
naming nit: we usually use lower case names for context managers, even though they're clases (e.g. `torch.cuda.device`).